public/ticketmanager.php

```diff
@@ -464,6 +594,9 @@
                 echo "<th>Game</th>";
                 echo "<th>Developer</th>";
                 echo "<th>Reporter</th>";
+                if ($closedTickets || $resolvedTickets) {
+                    echo "<th>Resolver</th>";
+                }
```

Comment:
Would it also make sense to hide the "Resolver" search field and "Resolved By: Not Author" selector if neither the closed or resolved subfilters are selected?

Reply:
I originally did this but these variables are retained so if you uncheck both Closed and Resolved but still have Resolver set it will not be visible but still affect the query. I opted to just always show them to not interfere too much with the users filter settings but the other option I considered was blanking out Resolver and unchecking Resolved By if both Closed and Resolved become unchecked. It would just be doing an extra check when unsetting Closed or Resolved to see if the other is unchecked and then modify the url to blank Resolver and uncheck Resolved By.

Reply:
You could just blank it out when reading the variables too.
```
        $resolvedByUser = requestInputSanitized('r', null);
        if ((!$closedTickets && !$resolvedTickets) || !isValidUsername($resolvedByUser)) {
            $resolvedByUser = null;
        }
```

Reply:
This came down to a similar situation as with the `$gamesTicketFilter` where even if I set `$resolvedByUser` before `$standardFilterUrl` based on closed and resolved status I would still need specific opposing Urls for the specific scenarios. Since these scenarios already take care of setting `$resolvedByUser` I didn't need to add the example provided. It now unsets those values when Closed and Resolved are not checked and hides Resolved By and Resolver sections.